packages/integrations/node/test/api-route.test.js

```diff
@@ -54,7 +55,7 @@ describe('API routes', () => {
 
 		let [out] = await done;
 		let arr = Array.from(new Uint8Array(out.buffer));
-		expect(arr).to.deep.equal([5, 4, 3, 2, 1]);
+		assert.notStrictEqual(arr, [5, 4, 3, 2, 1]);
```

Comment:
Should this be `assert.deepEqual` instead?

Reply:
Here's the weird thing, it doesn't pass the assertion

Reply:
just a heads up for knowledge sharing's sake, `strictEqual` compares references (hence why it failed with that error), and `deepEqual` from `node:assert` is legacy and has unexpected behaviours [as per the docs](https://nodejs.org/api/assert.html#assertdeepequalactual-expected-message).
You'd generally want to only use `node:assert/strict` where `deepEqual` is an alias for `deepStrictEqual`